src/utilities/nodeCache.js

```diff
@@ -0,0 +1,49 @@
+
+
+const NodeCache = require('node-cache');
+const logger = require('../startup/logger');
+
+const cacheStore = new NodeCache({ stdTTL: 100, checkperiod: 120 });
```

Comment:
Maybe have stdTTL as something larger than 100 seconds maybe set that to 300(every 5 mins) and checkperiod to 600(every 10 mins)